docker/Dockerfile

```diff
@@ -1,22 +1,17 @@
 FROM docker.io/golang:1.21rc3-alpine3.18
 
-RUN apk add --no-cache bash curl jq git sudo
+WORKDIR /etc/zoraxy/
 
-RUN mkdir -p /zoraxy/source/ &&\
-    mkdir -p /zoraxy/config/
+VOLUME [ "/etc/zoraxy/config/" ]
 
-VOLUME [ "/zoraxy/config/" ]
+COPY ./src ./
```

Comment:
Does this properly get the src directory from the repositories root? I don't have time to test it right now but this seems off to me

Reply:
Yes it does as long as you set the context to the root directory when you go to build the image